frontend/urls.py

```diff
@@ -1,6 +1,7 @@
-from django.urls import path
+from django.urls import re_path
+
 from . import views
 
 urlpatterns = [
-    path('', views.index)
-]
\ No newline at end of file
+    re_path("", views.index),

```

Comment:
What did this accomplish? And do we need to consider a converter to maintain standard use of `path`, which is the preferred use

Reply:
By changing this to `re_path`, all requests will be directed to the same view responding with the react app. Then react router can handle displaying the correct UI based on the URI

Reply:
It was just easier to match all URIs with a blank regex than some `path` type weirdness

Reply:
Awesome, I was hoping that was the case